commons-executors-api/build.gradle

```diff
@@ -5,3 +5,6 @@ libsDirName = file('build/artifacts')
 // Do not add dependencies to this project. This project is used by
 // api projects outside of atlasdb that try very hard to minimize
 // their dependency footprint.
+dependencies {
+    implementation 'com.boundary:high-scale-lib'
+}
```

Comment:
given the comment above, are we sure this is ok?

Reply:
I guess it's fine as this was implicitly added anyway, but is there a way to avoid this?